### DISCUSSION TITLE
Print with visible layers

### DISCUSSION THREAD

**Discussion by felipe-mendieta** [infra-geo-ouverte/igo2#770] [Q&A]
Hi, thanks for make igo open source is really useful, in my university are working in a open source and free chatbot for this plattaform.

I have this problem when I try print with a visible layer, only works when change from pdf to image but in image not is visible any layer, only show the map. In others version of igo2, the function print works but not are visible the layers too.
![image](https://user-images.githubusercontent.com/48095820/166061394-752a228d-715d-4a81-9d03-91de0e14fed7.png)

**Comment by pelord** [comment]
It's the same issue as the previous. you must handle your cors before ;)

**Reply by pelord** [reply]
Do you have any access to your server installation?

**Reply by felipe-mendieta** [reply]
No, I dont have access to an server, but I want to put maps from others sites. For example
https://www.geoportaligm.gob.ec/geoserver/mapabase/wms?service=wms&version=1.3.0&request=GetCapabilities

I need to have the layers in an own server if I want print a layer?
